ui/css/logviewer.css

```diff
@@ -53,6 +53,7 @@ body {
 
 .text-danger {
     color: #db0000;
+    background: #fbe3e3 !important;
```

Comment:
Agreed, not desirable. Afaict we need it to override the normal alternating white/grey line row colors applied by `.lv-log-container > div:nth-child(even)`, without which the grey backgrounded lines get precedence and those failure lines aren't painted correctly in pink, only the white ones.

I'll double check to be sure.